xml/Microsoft.Extensions.FileProviders.Physical/PhysicalFileInfo.xml

```diff
@@ -112,7 +112,8 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>
+          <see langword="true" /> if resource exists in the underlying storage system.</summary>
         <value>To be added.</value>
```

Comment:
Moving to the proper location:
```suggestion
        <summary>Determines if the resource exists in the underlying system.</summary>
        <value><see langword="true" /> if the resource exists in the underlying storage system; <see langword="false" /> otherwise.</value>
```

Reply:
@carlossanlop Do you usually click "Add single comment" or "Start a review" when you review the PRs? If you're using "add single comment", it sends us all an email each time you leave a comment. Any chance you can switch to the "Start a review" style, so it sends one email with all your comments together?